kotlin/src/main/com/looker/rtl/AuthToken.kt

```diff
@@ -32,9 +32,9 @@ data class AuthToken(
         @JsonProperty("access_token")
         var accessToken: String = "",  // TODO: Consider making this/these vals and using new objects instead of mutating
         @JsonProperty("token_type")
-        val tokenType: String = "",
+        var tokenType: String = "",
         @JsonProperty("expires_in")
-        var expiresIn: Long = 0,
+        var expiresIn: Long = 0L,
         @JsonProperty("refresh_token")
         var refreshToken: String? = "") {
```

Comment:
Is this default value = empty string going to interfere with the token.refresh_token != nil check below? or other places that need to know if the refresh_token is nil?

Reply:
I too think this default value is a bit strange. But the check below is for what the AccessToken has on it, which gets optionally set _into_ this field.

This default value could cause strange behavior in OAuthSession where this value is read, but I decided against making larger changes where it seems to be working.